### PR TITLE
Pump

### PR DESCRIPTION
**Pull request by TaylorSmith28**
Implemented the pump device.

### CODE REVIEW

File: titration/devices/syringe_pump.py
Comment (TaylorSmith28) [inline titration/devices/syringe_pump.py:1]: Change the __drive_pump_in and __drive_pump_out function names appropriately

File: titration/ui_state/demo_mode/demo_pump.py [hunk 1]
```diff
@@ -35,18 +36,25 @@ def handle_key(self, key):
...
             elif key == Keypad.KEY_2:
                 self._set_next_state(PumpVolume(self.titrator, self), True)
+                self.substate = 3
+
             elif key == Keypad.KEY_3:
-                # self.titrator.pump.pump_volume_in()
-                pass
+                self._set_next_state(VolumeToMove(self.titrator, self), True)
+                self.titrator.pump.pull_volume_in(self.titrator.volume_to_move)
+                self.titrator.volume_to_move = 0
```
Comment (prestoncarman) [inline titration/ui_state/demo_mode/demo_pump.py:47]: Does this go to the view volume state after collecting the input?

Comment (TaylorSmith28) [reply titration/ui_state/demo_mode/demo_pump.py:47]: It does now. This issue has been fixed.

**Review by prestoncarman — APPROVED** [review]
LGTM

**Pull request merged**